iOS Example Frame/Podfile.lock

```diff
@@ -1,10 +1,13 @@
 PODS:
-  - Alamofire (5.4.1)
+  - Alamofire (5.4.2)
```

Comment:
did we update this ?

Reply:
yes, the version is set to `~> 5.4.0` which means that any new patch version will be automatically updated to